wvanreeven/all_sky.ipynb

```diff
@@ -0,0 +1,87 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lsst.ts import salobj\n",
+    "domain = salobj.Domain()\n",
+    "all_sky = salobj.Remote(domain=domain, name=\"GenericCamera\", index=1)\n",
+    "print(dir(all_sky))\n",
+    "await all_sky.start_task\n",
+    "print(await all_sky.evt_heartbeat.next(flush=True, timeout=5))\n",
+    "all_sky.cmd_start.set(settingsToApply=\"all_fields\")\n",
+    "await all_sky.cmd_start.set_start(timeout=20)\n",
+    "await salobj.set_summary_state(remote=all_sky, state=salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await all_sky.cmd_takeImages.set_start(numImages=1, expTime=1.0, shutter=True, science=True, guide=False, wfs=False, imageSequenceName=\"test\")\n",
+    "endReadout = await all_sky.evt_endReadout.next(flush=False, timeout=20)\n",
+    "print(endReadout)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(remote=all_sky, state=salobj.State.DISABLED)"
```

Comment:
You know you can do all these state transitions in one go right?

```
await salobj.set_summary_state(remote=all_sky, state=salobj.State.OFFLINE)
```

Reply:
Yes I do and I will very likely change that at some point. At the moment I am still exploring all possibilities with both the notebooks and the CSC which is why I did it like this.